### ISSUE TITLE
I had to replace @ in order to compile with `acme` in Raspbian Buster Linux

### ISSUE THREAD

**Issue by mobluse** [indigodarkwolf/x16-matriculate-text#1]
I had to replace @ with ___ in the asm file in order to compile with `acme` in Raspbian Buster Linux.
I got my version of ACME from the distribution:
```
$ acme --version
This is ACME, release 0.96.2 ("Fenchurch"), 10 Mar 2017
  Platform independent version.
```
Now it runs!

**Comment by indigodarkwolf** [comment]
Pull request? Which @? All of them?

**Comment by indigodarkwolf** [comment]
I have had some problems with "@" in acme. I just don't understand what it's supposed to do differently from ".", and if you read my style-guide.txt, I even made a note to stop using them. Thought I'd got them all.

**Comment by mobluse** [comment]
> Pull request? Which @? All of them?

Yes, all of them, but there were not so many and they are in a small block of code. I replaced them with ___ (three underbars) because that wasn't used before. I thought this was a too minor change to send a pull request, but I could do if you like.

**Comment by indigodarkwolf** [comment]
No worries. I guess I hadn't really indicated that I'd be open to accepting pull requests. And as luck would have it, I was asked to update this for the new Vera 0.8, so I've gone ahead and taken care of it. They're not in the main branch *yet*, because I don't want to clobber the Vera 0.7 version until r31 officially releases it, but once r31 is out, I'll merge everything over.

**Comment by indigodarkwolf** [comment]
On second thought, I suppose keeping the Vera 0.7 version is what "Releases" are for, so I've gone ahead and created one for r30 and merged the Vera 0.8 branch, which should be free of the accursed '@' symbol.
